frame/contracts/src/weights.rs

```diff
@@ -202,11 +202,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts CodeStorage (r:1 w:1)
 	// Storage: Contracts PristineCode (r:0 w:1)
 	fn instantiate_with_code(c: u32, s: u32, ) -> Weight {
-		(499_349_000 as Weight)
-			// Standard Error: 199_000
-			.saturating_add((174_439_000 as Weight).saturating_mul(c as Weight))
-			// Standard Error: 13_000
-			.saturating_add((2_096_000 as Weight).saturating_mul(s as Weight))
+		(420_314_000 as Weight)
+			// Standard Error: 132_000
+			.saturating_add((170_494_000 as Weight).saturating_mul(c as Weight))
```

Comment:
I expected this to do down to 70us so that it matches the cost of `instrument`. My new suspicion is that loading the code into wasmi is responsible for the weight discrepancy.